e2e/tests/test_e2e.py

```diff
@@ -587,6 +587,27 @@ def verify_role():
             print('Operator log: {}'.format(k8s.get_operator_log()))
             raise
 
+    @timeout_decorator.timeout(TEST_TIMEOUT_SEC)
+    def test_cross_namespace_secrets(self):
+        '''
+            Test secrets in different namespace
+        '''
+        app_namespace = "appspace"
+        k8s = self.k8s
+        k8s.api.core_v1.create_namespace(app_namespace)
+        k8s.api.custom_objects_api.patch_namespaced_custom_object(
+            'acid.zalan.do', 'v1', 'default',
+            'postgresqls', 'acid-minimal-cluster',
+            {
+                'spec': {
+                    'users':{
+                        'appspace.db_user': [],
+                    }
+                }
+            })
+        self.eventuallyEqual(lambda: k8s.count_secrets_in_namespace(app_namespace),
+                             1, "Secret not created in user namespace")
```

Comment:
To  make it more informative, please log the name of the secret and the namespace